t/schema.t

```diff
@@ -0,0 +1,39 @@
+use strict;
+use warnings FATAL => 'utf8';
+use utf8;
+use open ':std', ':encoding(UTF-8)'; # force stdin, stdout, stderr into utf8
+
+use Data::UUID;
+use Path::Tiny;
+use Test::Conch;
+use Test::Deep;
+use Test::Memory::Cycle;
+use Test::More;
+use Test::Warnings ':all';
+use JSON::Validator;
+
+
+my $json_schema = JSON::Validator->new->schema('http://json-schema.org/draft-04/schema#')->schema->data;
+
+my $t = Test::Conch->new;
+$t->load_fixture('legacy_datacenter');
```

Comment:
You shouldn't need any fixtures for this test.

Reply:
Removed.